### PR TITLE
Funding Source Small Employers

### PR DESCRIPTION
**Pull request by samuel-dev-dba**
Funding Source Small Employers Acceptance Test Changes

### CODE REVIEW

File: src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Fully funded - Learning (TT1).feature [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 	Given the current processing period is 2
 	And the payments are for the current collection year
-	And a learner with LearnRefNumber learnref1 and Uln 10000 undertaking training with training provider 10000
+	And a learner is undertaking a training with a training provider
 
 	And the SFA contribution percentage is 100%
 
 	And the required payments component generates the following contract type 2 payable earnings:
-	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Period | ULN   | TransactionType | Amount |
+	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Delivery Period	 | ULN   | TransactionType | Amount |
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Fully funded - Learning (TT1).feature]: Tidy up table

File: src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Fully funded - Learning (TT1).feature [hunk 2]
```diff
@@ -21,7 +21,7 @@ Scenario: Contract Type 2 Learning payment
 
 	Then the payment source component will generate the following contract type 2 coinvested payments:
 
-	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Period | ULN   | TransactionType | FundingSource			| Amount |
+	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Delivery Period	 | ULN   | TransactionType | FundingSource			| Amount |
 	| learnref1      | 10000 | p1                     | 2      | 10000 | Learning (TT1)  | CoInvestedSfa (FS2)		| 600    |
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Fully funded - Learning (TT1).feature]: Tidy up table

File: src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Learning (TT1) only - multiple periods.feature [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 	And the payments are for the current collection year
-	And a learner with LearnRefNumber learnref1 and Uln 10000 undertaking training with training provider 10000
+	And a learner is undertaking a training with a training provider
 
 	And the SFA contribution percentage is 90%
 
 	And the required payments component generates the following contract type 2 payable earnings:
 
-	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Period | ULN   | TransactionType | Amount |
+	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Delivery Period	 | ULN   | TransactionType | Amount |
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Learning (TT1) only - multiple periods.feature]: Tidy up table

File: src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Learning (TT1) only - multiple periods.feature [hunk 4]
```diff
@@ -23,7 +23,7 @@ Scenario: Contract Type 2 Learning payment
 
 	Then the payment source component will generate the following contract type 2 coinvested payments:
 
-	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Period | ULN   | TransactionType | FundingSource			| Amount |
+	| LearnRefNumber | Ukprn | PriceEpisodeIdentifier | Delivery Period	 | ULN   | TransactionType | FundingSource			| Amount |
```
Comment (davehowlett) [inline src/SFA.DAS.Payments.FundingSource.AcceptanceTests/Tests/ACT2_TransTypes_1to3_Basic_Tests/Learning (TT1) only - multiple periods.feature]: Tidy up table

File: src/SFA.DAS.Payments.FundingSource.AcceptanceTests/App.config [hunk 5]
```diff
@@ -10,8 +10,9 @@
   </connectionStrings>
   <appSettings>
     <add key="EndpointName" value="sfa-das-payments-fundingsource-acceptancetests" />
+    <add key="NonLevyFundedPaymentsServiceEndpointName" value="sfa-das-payments-fundingsource-nonlevy" />
```
[the remark this answers is not in the document]
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.FundingSource.AcceptanceTests/App.config]: NonLevyFundedPaymentsServiceEndpointName is input into FundingSource for Queue expecting a message.  Acceptance Test queue name is the incoming message into the acceptance test handler.

File: src/SFA.DAS.Payments.FundingSource.AcceptanceTests/App.config [hunk 6]
```diff
@@ -10,8 +10,9 @@
...
     <add key="Environment" value="DEVELOPMENT" />
-    <add key="TimeToWait" value="00:00:30" />
+    <add key="TimeToWait" value="00:00:15" />
```
[the remark this answers is not in the document]
Comment (samuel-dev-dba) [reply src/SFA.DAS.Payments.FundingSource.AcceptanceTests/App.config]: Yes has it takes too long to wait for 30secs I guess it depends on user preference I will say with a fast PC 15secs is more than enough

**Pull request merged**